src/shared/utils/networks.ts

```diff
@@ -40,6 +40,12 @@ const networks: Network[] = [
     port: 51235,
     unls: ['vl.devnet.rippletest.net'],
   },
+  {
+    network: 'nft-dev',
+    entry: 'xls20-sandbox.rippletest.net',
+    port: 51235,
+    unls: ['nftvalidators.s3.us-west-2.amazonaws.com/index.json'],
```

Comment:
Where did you look to find this UNL?

Reply:
https://xls20-sandbox.rippletest.net:51235/crawl